docs/source/reference/packs.rst

```diff
@@ -84,140 +91,126 @@ The ``aliases`` folder contains Action Aliases. See :doc:`Action Alias </chatops
 
-5. Create an action alias. The sample action alias below aliases the greet action and makes it accessible from ChatOps.
+  Copy the following content to sensors/sensor1.py
 
-Copy the following content to aliases/alias1.yaml
+  .. literalinclude:: /../../st2/contrib/hello_st2/sensors/sensor1.py
 
-.. literalinclude:: /../../st2/contrib/hello_st2/aliases/alias1.yaml
+5. Create a rule. The sample rule below is triggered by event from the sensor and invokes the action from the samples above.
 
-6. Create a policy. The sample policy below limits concurrent operation of the greet action.
+  Copy the following content to rules/rule1.yaml
 
-Copy the following content to policies/policy1.yaml
+  .. literalinclude:: /../../st2/contrib/hello_st2/rules/rule1.yaml
 
-.. literalinclude:: /../../st2/contrib/hello_st2/policies/policy1.yaml
+6. Create an action alias. The sample action alias below aliases the greet action and makes it accessible from ChatOps.
 
-7. Install this pack manually. Your pack should be versioned in git for this step to work, or you can move the files to ``/opt/stackstorm/packs``. It's up to you to figure out an optimal workflow for editing and versioning your packs, but we recommend using git and installing from your repositories.
+  Copy the following content to aliases/alias1.yaml
 
-.. code-block:: bash
+  .. literalinclude:: /../../st2/contrib/hello_st2/aliases/alias1.yaml
 
-   # 1. If hello_st2 is on the same server.
-   st2 pack install ./hello_st2
+7. Create a policy. The sample policy below limits concurrent operation of the greet action.
 
-   # 2. If hello_st2 is hosted on github.
-   st2 pack install https://github.com/<your pack URL>
+  Copy the following content to policies/policy1.yaml
 
-   # 3. If you want to move the pack without git.
-   cp -R ./hello_st2 /opt/stackstorm/packs
+  .. literalinclude:: /../../st2/contrib/hello_st2/policies/policy1.yaml
 
-   # Reload the content.
-   st2 pack register hello_st2
+8. Install the pack. We encourage using git, if you do so, ``st2 pack`` will greatly simplify
+your pack management. However you can figure out your own workflow for editing and versioning packs;
+you'll be placing the files to ``/opt/stackstorm/packs`` and [re-]loading the content.
 
-Once you follow steps 1-7 you will have created your first pack. Commands like ``st2 action list``, ``st2 rule list`` and ``st2 trigger list`` will show you the loaded content. To check if the sensor triggering action is working, run ``st2 execution list``, there should be an entry for executing ``hello_st2.greet`` every minute.
+  7.1 Use git and ``pack install`` (**recommended**):
 
-Next steps would be to create an integration pack for you favorite tool or service that you would like to use with |st2|. Happy hacking!
+  .. code-block:: bash
 
+    # Get the code under git
+    cd hello_st2
+    git init && git add ./* && git commit -m "Initial commit"
+    # Install from local git repo
+    st2 pack install file:///$PWD
```

Comment:
I really don't think we should encourage this. We support local repos, and it's fine, but I think the model approach should be editing files on your laptop in your favorite text editor with linter plugins and such, then uploading to github, then doing `st2 pack install <github URL>=<your dev branch>` (because you would use a dev branch like a good person). 

Local repos help a lot in some cases, but in a "normal" st2 dev workflow (assuming there is such a thing) your st2 box should be a destination for the integration packs, not an origin.